### PR TITLE
翻译了 commands/web/

### PR DESCRIPTION
**Pull request by zhengjiani**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/cn/commands/web/execute.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+```
+
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands-yml/commands/web/execute.yml)
+## 描述
+
+假定执行的脚本是同步的，并且评估脚本的执行结果将返回给客户端。
+
+脚本参数定义了脚本以函数体形式执行， 该函数的返回值也将返回给客户端. 函数将调用提供的参数数组以及通过指定顺序获取的参数对象的值.
+
+参数可以是任何JSON原语, 数组, 或者 JSON 对象。 定义WebElement引用的JSON对象将转换为相应的DOM元素。 同样地, 脚本结果中的任何WebElements都将作为WebElement JSON对象返回给客户端。
```
Comment (lihuazhang) [inline docs/cn/commands/web/execute.md]: JSON-primitive 翻译成 JSON原语, 这个应该是 JSON 的基本类型

File: docs/cn/commands/web/execute.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+原生上下文（Native Context）中 `execute` , 详见 [Mobile Commands](/docs/en/commands/mobile-command.md).
```
Comment (lihuazhang) [inline docs/cn/commands/web/execute.md]: native context 这个就别翻译了

File: docs/cn/commands/web/execute.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+|  | [Espresso](/docs/en/drivers/android-espresso.md) | None | None | None |
+|  | [UiAutomator](/docs/en/drivers/android-uiautomator.md) | None | None | None |
+| Mac | [Mac](/docs/en/drivers/mac.md) | None | None | None |
+| Windows | [Windows](/docs/en/drivers/windows.md) | None | None | None |
+
+
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands-yml/commands/web/execute.yml)
+### Appium Clients
+
+|语言|支持|文档|
```
Comment (lihuazhang) [inline docs/cn/commands/web/execute.md]: 支持是支持的什么？

File: docs/cn/commands/web/navigation/url.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands-yml/commands/web/navigation/url.yml)
+# 获取URL
+
+检索当前页面的URL (仅是 Web context)
```
Comment (lihuazhang) [inline docs/cn/commands/web/navigation/url.md]: 获取当前页面的URL

**Review by lihuazhang — CHANGES_REQUESTED** [review]

File: docs/cn/commands/web/navigation/url.md (hunk 4 above)
Comment (zhengjiani) [reply docs/cn/commands/web/navigation/url.md]: 已全部做修改，谢谢指导～

File: docs/cn/commands/web/frame/switch.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands-yml/commands/web/frame/switch.yml)
+# 切换框架
+
+将焦点更改为页面上的另一个框架 (仅是 Web context)
```
Comment (lihuazhang) [inline docs/cn/commands/web/frame/switch.md]: frame 还是不要翻译成框架吧？

你搜索下，所有的frame，还是用frame吧，框架理解不了。大部分地方都不翻译这个字的

File: docs/cn/commands/web/navigation/back.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands-yml/commands/web/navigation/back.yml)
+# 回退
+
+如果可以，向后导航浏览器历史记录 (仅是 Web context)
```
Comment (lihuazhang) [inline docs/cn/commands/web/navigation/back.md]: 这句看不太懂，能不能调整下？比较顺口一点？

File: docs/cn/commands/web/navigation/forward.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands-yml/commands/web/navigation/forward.yml)
+# 前进
+
+如果可以，向前导航浏览器历史记录 (仅是 Web context)
```
Comment (lihuazhang) [inline docs/cn/commands/web/navigation/forward.md]: 这句也是，不能对着词语翻译，组织下中文语句。

**Review by lihuazhang — CHANGES_REQUESTED** [review]

File: docs/cn/commands/web/navigation/forward.md (hunk 7 above)
Comment (zhengjiani) [reply docs/cn/commands/web/navigation/forward.md]: 已修改+新提交，麻烦review～

File: docs/cn/commands/web/frame/parent.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+```csharp
+// C#
+driver.SwitchTo().ParentFrame();
+
+```
+
+[//]: # (DO NOT EDIT THIS FILE! This is an auto-generated file. Editing for this document happens in /commands-yml/commands/web/frame/parent.yml)
+## 描述
+
+如果当前上下文是顶级浏览上下文, 则上下文保持不变。
```
Comment (lihuazhang) [inline docs/cn/commands/web/frame/parent.md:53]: 顶级浏览上下文 这个看不太懂诶

**Review by lihuazhang — APPROVED** [review]

**Pull request merged**